airflow/runtimes/container_runtime.go

```diff
@@ -1,17 +1,59 @@
-package airflow
+package runtimes
 
 import (
 	"os"
 	"path/filepath"
 	"strings"
 	"sync"
+	"time"
+
+	"github.com/briandowns/spinner"
 
 	"github.com/astronomer/astro-cli/config"
 	"github.com/astronomer/astro-cli/pkg/fileutil"
 	"github.com/astronomer/astro-cli/pkg/util"
 	"github.com/pkg/errors"
 )
 
+var spinnerCharSet = spinner.CharSets[14]
```

Comment:
Do we maybe want to match (or reuse? or change?) the charset in `pkg/ansi/spinner.go` for consistency?

Reply:
Looks like we created a custom one for login, etc. I think standardizing would be good but I'm trying to keep my changes to a minimum and keep them highly correlated. I've started and stopped this PR a few times because I've gone too far down the refactor path. I do plan to keep going, and intend to standardize as much as possible as we go it'll just be in targeted chunks.

This particular spinner charset actually mimics the ones that docker-compose uses so it looks somewhat intentional at the moment since we're using it for the messages where we're dealing with containers and the runtimes, etc.

![Screenshot 2024-11-25 at 1 14 50 PM](https://github.com/user-attachments/assets/b8b52abe-3edc-48cf-b92e-d6c02f71f655)

Reply:
Ok fair enough. I appreciate the refactor considering it can feel like a bit of a never-ending rabbit hole.